src/dune

```diff
@@ -0,0 +1,2 @@
+(executable
+  (name generate_letters))
```

Comment:
The letter generator needs to be in the `/lib` folder, not a separate executable